src/main/java/com/staffmanagement/dto/UserDTO.java

```diff
@@ -0,0 +1,45 @@
+package com.staffmanagement.dto;
+
+public class UserDTO {
+
+	private String name;
+
+	private String profession;
+
+	private String age;
+
+	private String address;
+
+	public String getName() {
```

Comment:
instead of writing getter setter, i would suggest using [lombook](https://projectlombok.org/)